modules/statics/src/coins.ts

```diff
@@ -3113,6 +3113,14 @@ export const coins = CoinMap.fromCoins([
     '0xf7790914dc335b20aa19d7c9c9171e14e278a134',
     UnderlyingAsset.EURCV
   ),
+  erc20(
+    '4a47b0a6-f5a4-4a74-89f0-2475fb92e191',
+    'eurcvv2',
+    'EUR CoinVertible',
+    18,
+    '0x5f7827fdeb7c20b443265fc2f40845b715385ff2',
```

Comment:
https://etherscan.io/address/0x5f7827fdeb7c20b443265fc2f40845b715385ff2 shows this as `eurcv` not `eurcvv2`. How will a user know this is this specific contract in places like the UI?

Reply:
the "v2" is our bitgo platform label which is the latest version (on chain they jsut call it the symbol since the older contract is just deprecated but we need to have it in bitgo since tehcnialyl they are separate tokens)

Reply:
actually @mmcshinsky-bitgo we're going to make following change
change existing EURCV -> EURCVv0 (since no one is using the old token)
EURCVv2 -> EURCV